.gitignore

```diff
@@ -29,6 +29,10 @@
 /.env*
 /apps/joiner
 /deps/deno
+/data/sql/custom/*
+/src/server/scripts/Custom/*
+!/src/server/scripts/Custom/custom_script_loader.cpp
```

Comment:
this i am not feeling right about with the script loader.

Reply:
Good eye, I thought it was static. I'll be able to get that fixed in a little bit

Reply:
@acidmanifesto This should be fixed.